### DISCUSSION TITLE
support coder-logs location setting?

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#5933] [Q&A]
I am curious any CLI-arg to setting coder-logs? I found no-arg related it.

now, logs location like the followings.
<img width="341" alt="image" src="https://user-images.githubusercontent.com/9245110/210036692-029edcd6-05bd-40b8-9544-3d6df16266ea.png">

any issue can lead to this. when I run multi times, logs be multi .like the followings.

I don't know the cause of the followings. maybe code-server, or supervisor

<img width="557" alt="image" src="https://user-images.githubusercontent.com/9245110/210036892-29e89c2f-e68d-4e9c-8b7b-06ffdfb20824.png">

## version info
1. v4.7.1
2. process controller is supervisor
```
[program:code-server]
command=%(ENV_CODE_SERVER_DIR)s/bin/code-server --bind-addr=%(ENV_BIND_ADDR)s --user-data-dir=%(ENV_USER_DATA_PATH)s --config %(ENV_CONFIG_PATH)s  --disable-update-check --locale=zh-cn --extensions-dir=%(ENV_EXTENSION_PATH)s
autostart=false
autorestart=true
stopwaitsecs=3
stdout_logfile=%(ENV_CODE_SERVER_RUN_DIR)s/supervisord-conf/code-server.log
redirect_stderr=true
```

**Comment by jsjoeio** [comment]
I don't think we support that 🤔 @code-asher might know though if there's history around that.

**Reply by code-asher** [reply]
Yeah, no configuration at the moment for the log location (other than changing the data directory entirely).

I have seen the bug where code-server creates a completely new path with `<date>/full/path/to/.local/share/code-server/coder-logs` in the working directory instead of using `~/.local/share/code-server/coder-logs` but I have not been able to reliably reproduce it.

I wonder if we should just remove the file-based logs and only use stdout.  Then users can log that to wherever they want.
